### ISSUE TITLE
possible missing runtime dependency for borg

### ISSUE THREAD

**Issue by paoloschi** [voidlinux/void-packages#4352]
just tried the backup application **borg**
at the first run I got:
```
$ borg init test-backup
Traceback (most recent call last):
  File "/usr/bin/borg", line 5, in <module>
    from pkg_resources import load_entry_point
ImportError: No module named 'pkg_resources'
```
I could solve the problem by installing **python3.4-setuptools** therefore this package should probably be included among the automatic **borg** runtime dependencies (?)

**Comment by chneukirchen** [comment]
yes

**Comment by chneukirchen** [comment]
Fixed by e47e425
